docs/learn.rst

```diff
@@ -120,20 +120,49 @@ Search
 
 If you are looking for a specific subject, topic, or term, use the **Search** feature:
 
-#. Click the magnifying glass icon in the upper right corner.
-#. Type the word or combination of words you are looking for in the search field.
-#. Press **Enter** to display search results below the field.
+#. Type the word or combination of words you are looking for in the search field located in the top bar.
+#. Press **Enter** or click the |search| button to display the results.
 
 .. figure:: img/search.png
-	:alt: search page
+	:alt: Filter the search results by channel or type of content.
+
+	Filter the search results by channel or type of content.
+
+
+Some Kolibri resources may be duplicated in different topics and/or channels. Search results will indicate when a resource is duplicated, and you can click on the ``...locations`` link to discover the details for each location.
+
+.. figure:: img/search-duplicates.png
+	:alt: Duplicated resources in the search results.
+
+	Duplicated resources in the search results.
+
+
+.. figure:: img/search-duplicates2.png
+	:alt: Details for the duplicated resources in the search results.
+
+	Details for the duplicated resources in the search results.
 
-	Search page gives you option to search for a specific term in Kolibri.
 
 .. _content_types:
 
 Content types
 ^^^^^^^^^^^^^
 
+You may be different types of content available in Kolibri channels, and you can easily distinguish them by their icons:
+
+* Videos - |video-icon|
+* Exercises - |exercise-icon|
+* Documents - |doc-icon|
+* Interactive HTML Apps - |html-icon|
+* Topics - |topic-icon|
+
+When you start doing an exercise, viewing a video, or reading a document, it will be marked with the |in-progress| icon. Once you finish or complete them, they will be marked with the |completed| icon.
```

Comment:
This paragraph switches from singular (an exercise, it) to plural (they will be marked)